contrib/mujs/patches/read_env_cflags.patch

```diff
@@ -0,0 +1,69 @@
+--- Makefile	2024-01-27 14:08:36.007610489 +0100
++++ Makefile	2024-01-27 18:07:07.795691322 +0100
+@@ -4,10 +4,10 @@
+ 
+ build ?= release
+ 
+-prefix ?= /usr/local
+-bindir ?= $(prefix)/bin
+-incdir ?= $(prefix)/include
+-libdir ?= $(prefix)/lib
++PREFIX ?= /usr/local
++BINDIR ?= $(PREFIX)/bin
++INCDIR ?= $(PREFIX)/include
++LIBDIR ?= $(PREFIX)/lib
+ 
+ ifeq "$(wildcard .git)" ".git"
+   VERSION := $(shell git describe --tags --always)
+@@ -23,7 +23,7 @@
+ 
+ # Compiler flags for various configurations:
+ 
+-CFLAGS := -std=c99 -pedantic -Wall -Wextra -Wno-unused-parameter
++CFLAGS := -std=c99 -pedantic -Wall -Wextra -Wno-unused-parameter $(CFLAGS)
+ 
+ ifeq "$(CC)" "clang"
+   CFLAGS += -Wunreachable-code
+@@ -115,28 +115,28 @@
+ 	@ while ! inotifywait -q -e modify $(SRCS) $(HDRS) ; do time -p $(MAKE) ; done
+ 
+ install-common: $(OUT)/mujs $(OUT)/mujs.pc
+-	install -d $(DESTDIR)$(incdir)
+-	install -d $(DESTDIR)$(libdir)
+-	install -d $(DESTDIR)$(libdir)/pkgconfig
+-	install -d $(DESTDIR)$(bindir)
+-	install -m 644 mujs.h $(DESTDIR)$(incdir)
+-	install -m 644 $(OUT)/mujs.pc $(DESTDIR)$(libdir)/pkgconfig
+-	install -m 755 $(OUT)/mujs $(DESTDIR)$(bindir)
++	install -d $(DESTDIR)$(INCDIR)
++	install -d $(DESTDIR)$(LIBDIR)
++	install -d $(DESTDIR)$(LIBDIR)/pkgconfig
++	install -d $(DESTDIR)$(BINDIR)
++	install -m 644 mujs.h $(DESTDIR)$(INCDIR)
++	install -m 644 $(OUT)/mujs.pc $(DESTDIR)$(LIBDIR)/pkgconfig
++	install -m 755 $(OUT)/mujs $(DESTDIR)$(BINDIR)
```

Comment:
all of these changes are useless, all you need is to pass prefix=/usr and then nothing has to be changed